src/expmv.jl

```diff
@@ -2,11 +2,11 @@ export expmv, expmv!
 
 function default_anorm(A)
     try
-        Compat.opnorm(A, Inf)
+        opnorm(A, Inf)
     catch err
         if err isa MethodError
-            warn("opnorm($(typeof(A)), Inf) is not defined, fall back to using `anorm = 1.0`.
-To suppress this warning, please specify the anorm parameter manually.")
+            @warn "opnorm($(typeof(A)), Inf) is not defined, fall back to using `anorm = 1.0`.
+To suppress this warning, please specify the anorm keyword manually."
```

Comment:
unrelated to 1.0 compatibility, but the second mention of `anorm` could have backquotes around it